sdk/eventhub/azure-eventhub/HISTORY.md

```diff
@@ -1,10 +1,26 @@
 # Release History
 
-## 5.0.0b7 (Unreleased)
+## 5.0.0 (2020-01-06)
 
 **Breaking changes**
 
-- Removed deprecated property EventData.application_properties and deprecated method EventData.encode_message()
+- `EventData`
+    - Removed deprecated property `application_properties` and deprecated method `encode_message()`
+- `EventHubConsumerClient`
+    - `on_error` would be called when `EventHubConsumerClient` failed to claim ownership of partitions.
+    - `on_partition_cloe` and `on_partition_initialize` would be called in the case of meeting exceptions raised by callback `on_event`.
```

Comment:
Typo: `on_partition_close`
I would also change "meeting exceptions" to just "in the case of exceptions raised by `on_event` callback."

Reply:
updated